.gitignore

```diff
@@ -12,3 +12,4 @@ system-test/*key.json
 .DS_Store
 package-lock.json
 __pycache__
+.vscode
```

Comment:
we should consider moving `.gitignore` into a template, @sofisl does your library generation tool populate the file now?